hack/generate-groups.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
what's the take on the license headers? In particular the copyright statement? The codebase is really all over the place. Personally I am not a fan of adding the license header for each file. AFAIU the Apache License just requires the LICENSE to be added once. 

I am in particular wondering about the copyright statement.

Reply:
Copy-pasted from Tekton's (same reason for the `update-codegen.sh` one), and I tend to not remove copyright headers. I'll remove the license, though, since it's unnecessary.

Reply:
ok.